datacenterlight/locale/de/LC_MESSAGES/django.po

```diff
@@ -516,6 +516,13 @@ msgstr "Ungültige Speicher-Grösse"
 msgid "Incorrect pricing name. Please contact support{support_email}"
 msgstr ""
```

Comment:
Context?

Reply:
Pricing name identifies what pricing is applied in the calculator. It is something that is known to the admins only and nothing to do with end user. As we will show this message to the end user, I think we should rephrase this error. What do you say @verysanghee ?